naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/GraphqlSchemaProvider.scala

```diff
@@ -97,6 +110,10 @@ class DefaultGraphqlSchemaProvider @Inject() (schemaProvider: SchemaProvider)
     checkSchema()
     cachedSchema
   }
+
+  override def errors: SchemaErrors = {
+    schemaErrors
```

Comment:
I'm not aware of the standard around this but I've often seen these 'getter' like functions written as 

```
override def errors: SchemaErrors = schemaErrors
```

not sure what the practice around this is.